src/main/java/core/basesyntax/strategy/BalanceHandler.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.FruitTransaction;
+import core.basesyntax.Storage;
+
+public class BalanceHandler implements OperationHandler {
+    @Override
+    public void toStorage(FruitTransaction transaction) {
```

Comment:
do not forget common mistakes suggestions:
"Check result balance before saving it in the Storage - it should be positive. Throw RuntimeException in case the balance is negative." in all cases where you put your result in the storage